lib/CustomFields/ApiDataSanitizer.php

```diff
@@ -20,16 +20,16 @@ function sanitize(array $data = []) {
 
   private function checkMandatoryStringParameter(array $data, $parameter_name) {
     if (empty($data[$parameter_name])) {
-      throw new InvalidArgumentException(sprintf(__('Mandatory argument "%s" is missing', 'mailpoet'), $parameter_name));
+      throw new InvalidArgumentException(sprintf(__('Mandatory argument "%s" is missing', 'mailpoet'), $parameter_name), 1001);
```

Comment:
The docs don't mention that an `InvalidArgumentException` can be thrown, or was this implied by the "exception of base class `\Exception`" line?

Reply:
This error is caught in the main MP API class and converted to `APIException` so we don't need to mention it.